### PR TITLE
issue #1: first working version sending email and updating status

### PR DESCRIPTION
**Pull request by ancagabriela**
issue #2

### CODE REVIEW

File: gas/sendEmail.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+// Use this code for Google Docs, Slides, Forms, or Sheets.
+function onOpen() {
+  SpreadsheetApp.getUi() // Or DocumentApp or SlidesApp or FormApp.
+      .createMenu('Coderbunker')
+      .addItem('Send email', 'sendEmail')
+      .addToUi();
+}
+
+function sendEmail() {
+  var file = DriveApp.getFileById('1WKqYD9GLqmLjDqJyeY4BPkvl31vgRAeM');
```
Comment (rngadam) [inline gas/sendEmail.js]: move constants into Script Properties

File: gas/sendEmail.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  
+  var spreadsheet = SpreadsheetApp.getActiveSpreadsheet();
+  var sheet = spreadsheet.getSheetByName("Emails");
+  var startRow = 2; // First row of data to process
```
Comment (rngadam) [inline gas/sendEmail.js]: instead of redundant comment, make your variables self-descriptive:

firstRowOfDataToProcess

File: gas/sendEmail.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  var numRows = sheet.getLastRow(); // Number of rows to process
```
Comment (rngadam) [inline gas/sendEmail.js]: numRowsToProcess

File: gas/sendEmail.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+function sendEmail() {
+  var file = DriveApp.getFileById('1WKqYD9GLqmLjDqJyeY4BPkvl31vgRAeM');
+  
+  var spreadsheet = SpreadsheetApp.getActiveSpreadsheet();
+  var sheet = spreadsheet.getSheetByName("Emails");
+  var startRow = 2; // First row of data to process
+  var numRows = sheet.getLastRow(); // Number of rows to process 
+  // Fetch the range of cells (row, column, numRows, numColumns--this check column 3 is it is duplicate)
+  var dataRange = sheet.getRange(startRow, 1, numRows - 1, 3);
```
Comment (rngadam) [inline gas/sendEmail.js]:
````
const SENT_COLUMN_INDEX = 3;

const dataRange = sheet.getRange(startRow, 1, numRows - 1, SENT_COLUMN_INDEX);
```

for variables you don't expect to be reassigned use const

File: gas/sendEmail.js [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+  // Fetch values for each row in the Range.
+  var data = dataRange.getValues();
+  for (var i = 0; i < data.length; ++i) {
+    var row = data[i];
+    var name = row[0]; // First column
+    var email = row[1]; // Second column
+    var isBlank = sheet.getRange(startRow + i, 3, 1, 1).isBlank();
```
Comment (rngadam) [inline gas/sendEmail.js]:
```
var isBlank = sheet.getRange(startRow + i, SENT_COLUMN_INDEX, 1, 1).isBlank();
```

File: gas/sendEmail.js [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+    if (isBlank) { // Prevents sending duplicates
+      var html = HtmlService.createHtmlOutputFromFile('emailTemplate').getContent();
```
Comment (rngadam) [inline gas/sendEmail.js]: var html = HtmlService.createHtmlOutputFromFile('emailTemplate').getContent();

should eventually be

```
let htmlTemplate = HtmlService.createHtmlOutputFromFile('emailTemplate');
let htmlRendered = htmlTemplate.evaluate();
let htmlContent = htmlRendered.getContent();
```

File: gas/sendEmail.js [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+      var subject = name + "'s " + "Coderbunker Onboarding";
+  
+      GmailApp.sendEmail(email, subject, "",
```
Comment (rngadam) [inline gas/sendEmail.js]:
```suggestion
      GmailApp.sendEmail(
          email, 
          subject, 
          '', 
          {
            from: 'services@coderbunker.com', 
            name: 'Coderbunker Services',
            cc: 'bizdev@coderbunker.com',
            htmlBody: html,
            inlineImages: { 
              imageKey: file.getAs(MimeType.PNG) 
            }
          }
      );
```

**Comment by rngadam** [comment]
@ancagabriela when will this be done?

File: gas/sendEmail.js (hunk 6 above)
Comment (ancagabriela) [reply gas/sendEmail.js]: that is not working when I try to change it

Comment (ancagabriela) [reply gas/sendEmail.js]: but it works without:
let htmlRendered = htmlTemplate.evaluate();

**Pull request merged**